lambda_functions/alerter/alerter.py

```diff
@@ -32,7 +32,7 @@ def lambda_handler(event, context):
     alert_message = {
         "Summary": f"The website at {url} is unreachable. Healthcheck failure.",
         "Alarm": "Route53 Health Check Failure.",
-        "Description": f"Website is unreachable or not returning the expected response. Check Amazon Cloudwatch Alarms \'{alarm_name}\' and Healthcheck Lambda \'{lambda_name}\'.",
+        "Description": f"Website is unreachable or not returning the expected response. Check Amazon Cloudwatch Alarms '{alarm_name}' and Healthcheck Lambda '{lambda_name}'.",
```

Comment:
This continually gets changed by Black - we need to add something so that black doesn't make the change. James is looking at it - but can you revert the change here please.

Reply:
Pushed up the changes now